packages/account/src/Sections/Security/AccountClosed/account-closed.tsx

```diff
@@ -20,7 +23,7 @@ const AccountClosed = ({ logout }) => {
     }, [is_appstore, timer]);
 
     React.useEffect(() => {
-        history.push(routes.root);
+        window.history.pushState(null, '', '/');
```

Comment:
this will cause the page to reload , right? :thinking: 
but as I checked the attached recording video in the card, we should not reload the page when redirecting to the root page .
@jim-deriv correct me if I'm wrong please :pray:

Reply:
I just reverted some of the changes in this file, we were using this approach